kahuna/public/js/components/gr-export-original-image/gr-export-original-image.html

```diff
@@ -0,0 +1,14 @@
+<a class="drop-menu inner-clickable clickable side-padded button-right-side" ng:click="showUserActions = !showUserActions">
+    <span>▾</span>
+</a>
+
+<div ng:if="showUserActions"
+     class="drop-menu__items drop-menu__items--nopad"
+     ng:class="{ 'drop-menu__items--action' : ctrl.cropping }"
+     ng:click="ctrl.callCrop()">
+    <ul>
```

Comment:
> Or is it better to leave out and add in later if needed?

Generally, its better to do just what's needed at the time as there's (currently) no guarantee of more items being added.

Your call. :smile: